### PR TITLE
[IMP] mail: add quick reaction menu

### PR DESCRIPTION
**Pull request by tsm-odoo**
Before this PR, reacting to a message could be cumbersome especially for emojis that are frequently used. You first had to open the emoji picker then scroll to the top to the most frequent session to finally click on the desired emoji.

This PR introduces a quick reaction menu to allow quick reactions. The distance the user needs to scroll before reacting is much smaller, much closer to the message.

This is a first step toward message actions improvements. This is especially necessary for chat windows as the space left for the bubbles is too small, resulting in a poor experience.

task-4243619

| Before | After |
| ------------- | ------------- |
|   chrome-capture-2024-11-6 |   chrome-capture-2024-11-7  |

### CODE REVIEW

File: addons/web/static/src/core/frequent_emoji_service.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { reactive } from "@odoo/owl";
```
Comment (aab-odoo) [inline addons/web/static/src/core/frequent_emoji_service.js:1]: suggestion: create a folder /core/emojis/ to gather emojis stuff

Comment (aab-odoo) [reply addons/web/static/src/core/frequent_emoji_service.js:1]: side question: should this be in web? I guess it is for a reason, but who knows

Comment (tsm-odoo) [reply addons/web/static/src/core/frequent_emoji_service.js:1]: I'll create the folder. The emoji picker is in web and keeps track of the frequent emojis so it should be in web as well.

image

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by tsm-odoo** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Comment by tsm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**